react-native/READEME.md

```diff
@@ -0,0 +1,26 @@
+# React Native
+
+- Use React following the [React Guide](/react/)
+- Use [TypeScript](/typescript/)
+- Prefer using [core components and apis] over writing bespoke components.
+- Prefer grouping logic by feature rather than implementation detail.
```

Comment:
Would you be able to expand on this?

Reply:
Yes, but not succinctly. I think it makes sense to remove this line for now and reintroduce this in its own PR after some word smithing.